client/my-sites/site-settings/jetpack-site-stats.jsx

```diff
@@ -115,34 +113,34 @@ class JetpackSiteStats extends Component {
 		);
 	}
 
-	renderModuleEnableBanner() {
-		const { translate } = this.props;
+	render() {
+		const {
+			activatingStatsModule,
+			moduleUnavailable,
+			siteId,
+			siteRoles,
+			siteSlug,
+			translate,
+		} = this.props;
 
-		return (
-			<Banner
-				title={ translate( 'Site Stats module is disabled' ) }
-				description={ translate(
-					'Enable this to see detailed information about your traffic, likes, comments, and subscribers.'
+		const header = (
+			<JetpackModuleToggle
+				siteId={ siteId }
+				moduleSlug="stats"
+				label={ translate(
+					'See detailed information about your traffic, likes, comments, and subscribers.'
```

Comment:
I would really love it if we could keep this on a single line. Maybe `See details` instead of `See detailed information`, or `Monitor` instead of `See detailed information about`?